sql/catalyst/src/main/scala/org/apache/spark/sql/internal/SQLConf.scala

```diff
@@ -1749,6 +1749,16 @@ object SQLConf {
     .checkValue(v => v > 0, "The min partition number must be a positive integer.")
     .createOptional
 
+  val FILES_MAX_PARTITION_NUM = buildConf("spark.sql.files.maxPartitionNum")
```

Comment:
Could you add this new configuration to `sql-performance-tuning.md` like `spark.sql.files.minPartitionNum`?

https://github.com/apache/spark/blob/daee47a436473b865343301a78fef715d2780c4d/docs/sql-performance-tuning.md?plain=1#L89

Reply:
+1